src/parse_args.ts

```diff
@@ -60,7 +60,7 @@ program
     'pause on error until a keypress is made in the console. Useful during development'
   )
   .option('--screenshots', 'take screenshot for each step')
-  .option('--filmstrips', 'record detailed filmstrip info for all journeys')
+  .option('--trace', 'record chrome trace events for all journeys')
```

Comment:
we might need a way to disable filmstrips alone as it's going to increase the doc size to a larger extent.

Reply:
Why not keep the filmstrips arg and add the `--trace` arg to separate that then?

Reply:
Hmmm was afraid of introducing more flags. Let me have a look at the commander library for 2 level objects like trace.filmstrips and `trace-perf` as such.

Reply:
We will keep both trace and filmstrips and address the heartbeat config handling in https://github.com/elastic/synthetics/issues/289

Reply:
I think we have a solution for the too many flags issue in #289